libs/table/src/lib/pipes/aria-sort/aria-sort.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { NgxAbstractTableCellDirective } from '../../cell';
+import { NgxTableSortEvent } from '../../interfaces';
+
+@Pipe({
+	name: 'ngxAriaSort',
+	standalone: true,
+})
+export class NgxAriaSortPipe implements PipeTransform {
+	transform(value: {
+		currentSorting: NgxTableSortEvent;
+		cell: NgxAbstractTableCellDirective;
+	}): 'none' | 'ascending' | 'descending' {
+		const { cell } = value;
+
+		if (!cell || !cell.sortDirection) {
+			return 'none';
+		}
+
+		return cell.sortDirection.toLocaleLowerCase() as 'ascending' | 'descending';
```

Comment:
Maybe put this union type in the tree-grid types, for convenience?

Reply:
The reason why I didn't is because it's not really a type worth sharing everywhere. I prefer to keep these string types in the component/util/etc. where it's used if it has no other value throughout the application/lib.